clang/include/clang/Basic/Attr.td

```diff
@@ -1226,6 +1226,17 @@ def SYCLKernel : InheritableAttr {
   let Documentation = [SYCLKernelDocs];
 }
 
+def SYCLSpecialClass: InheritableAttr {
+  let Spellings = [Clang<"sycl_special_class">];
+  let Subjects = SubjectList<[CXXRecord]>;
+  let LangOpts = [SYCLIsDevice, SYCLIsHost];
+  let Documentation = [Undocumented];
+  let Args = [EnumArgument<"SpecialClassKind", "SpecialClassKind",
+                           ["accessor", "sampler", "stream", ""],
+                           ["Accessor", "Sampler", "Stream", "Generic"], 1>];
+  let PragmaAttributeSupport = 0;
```

Comment:
Is there a reason for disabling the usual pragma support (this is for `#pragma clang attribute` support, not support for the attribute as a pragma directly)?

Reply:
Not sure about this. @erichkeane?

Reply:
If it's an internal-only attribute, I suppose it doesn't much matter one way or the other. I'd say remove the line so it's more like "typical" attributes, but I don't feel strongly given that it's an internal one.

Reply:
I think this patch was originally written when we misunderstood what this meant. All SYCL attributes a one point, had let `PragmaAttributeSupport = 0`. This can be removed.